enterprise/server/remote_execution/execution_server/execution_server.go

```diff
@@ -883,29 +883,11 @@ func (s *ExecutionServer) PublishOperation(stream repb.Execution_PublishOperatio
 	if err != nil {
 		return err
 	}
-	lastOp := &longrunning.Operation{}
-	lastWrite := time.Now()
+	var lastOp *longrunning.Operation
 	taskID := ""
 	// Once the executor has called PublishOperation, we're in EXECUTING stage.
 	stage := repb.ExecutionStage_EXECUTING
 	mu := sync.Mutex{}
```

Comment:
nit: I think the mutex isn't needed anymore since we don't have any concurrent calls to `updateExecution` in this func?